### PR TITLE
Add PyLadies Praha Autumn (CZ.NIC + NTK)

### PR DESCRIPTION
**Pull request by hroncok**
The first 2 commits may be squashed, adding here, so @messa knows a bit how base works. (The result after first and after second commit is the same).

### CODE REVIEW

**Comment by hroncok** [comment]
Now it can be entirely squashed. @messa, could you please review this?

File: runs/2017/pyladies-praha-podzim-cznic/info.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    url: '#'
+  - title: Klondike Solitaire – hra na procvičení seznamů a n-tic
+    type: lesson
+    url: http://pyladies.cz/v1/s007-cards/klondike.html
+  - title:  Slovníky ve funkci popis_karty ve hře Klondike
+    type: lesson
+    url: http://pyladies.cz/v1/s011-dicts/klondike-dictget.html
+  - title: MicroPython a malé počítače
+    type: lesson
+    url: '#'
```
Comment (hroncok) [inline runs/2017/pyladies-praha-podzim-cznic/info.yml]: See https://github.com/pyvec/naucse.python.cz/issues/207 for explanation.

**Comment by messa** [comment]
@hroncok  Za mě dobrý :+1:

File: runs/2017/pyladies-praha-podzim-cznic/info.yml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  - +merge
+  - title: Domácí projekty (PDF)
+    url: http://pyladies.cz/v1/s004-strings/handout/handout4.pdf
+    type: homework
+- title: Procvičování
+  date: 2017-10-24
+  slug: exercises
+  materials: []
+- base: tests
+  slug: testing
```
Comment (hroncok) [inline runs/2017/pyladies-praha-podzim-cznic/info.yml]: This overrides the slug form the base (`test` -> `testing`). I'd rather get rid of this, but I kept it, so the links are consistent with spring run.

**Pull request merged**